### DISCUSSION TITLE
What is best flow to create an organization account?

### DISCUSSION THREAD

**Discussion by sawadashota** [ory/kratos#1958] [Q&A] [answered]
Hi,

I'm building ID system with ORY Kratos. The system has 2 types of accounts which are personal and organization accounts.
Organization accounts is created by organization admin via Kratos Admin API through my backend application but I'm struggling how to activate and verify email for new organization account.

Creating organization account flow in my system is following:

1. An organization admin creates new organization account via Kratos Admin API through my application.
2. My Application send email with recovery page to new account email because Identity Create API does NOT send email.
3. New user click the link in email. Then recovery flow initiated.
4. **The user received email again** and click link in email. Then recovery flow is completed and setting flow is started.
5. New user set password in setting page.

In this flow, system send email twice and users must click links in email twice.

Another flow:

1. An organization admin creates new organization account via Kratos Admin API through my application.
2. My Application send email with recovery link to new account email because Identity Create API does NOT send email.
3. New user click the link in email. Then recovery flow is completed and setting flow is started.
4. New user set password in setting page.
5. **Kratos send email for email verification.**

In this flow, system also send email twice and users must click links in email twice.

Then, what is best flow to create such an organization account?

**Comment by vinckr** [comment, marked as answer]
Hello, 
apologies for the late answer. 
Both approaches have the disadvantage of requiring the user to click two emails, right. 
You could skip the recovery flow (or rather let it complete in the background without user interaction); not sure if that has security implications. Then you only have one email in the second flow approach. 
From my POV I would recommend this, but maybe @aeneasr can also chime in to confirm 🙏

**Reply by vinckr** [reply]
Hello, 
apologies for reviving this old discussion, 
but did you find a good solution? 
I will mark it as answered to put it off the backlog, but feel free to come back to this anytime.
